cypress/support/custom_commands/login.helper.js

```diff
@@ -125,43 +125,8 @@ const fillLoginForm = (username, password) => {
 	);
 };
 
-const shuffleString = (str) => {
-	const characters = str.split("");
-	let currentIndex = characters.length;
-	let randomIndex;
-
-	while (currentIndex !== 0) {
-		randomIndex = Math.floor(Math.random() * currentIndex);
-		currentIndex--;
-		[characters[currentIndex], characters[randomIndex]] = [
-			characters[randomIndex],
-			characters[currentIndex],
-		];
-	}
-
-	return characters.join("");
-};
-const generateStrongPassword = (length) => {
-	const lowercase = "abcdefghijklmnopqrstuvwxyz";
-	const uppercase = lowercase.toUpperCase();
-	const numbers = "0123456789";
-	const symbols = "!@#$%^&*";
-
-	const allChars = lowercase + uppercase + numbers + symbols;
-
-	let password = "";
-	for (let i = 0; i < length; i++) {
-		const charSet = allChars[Math.floor(Math.random() * allChars.length)];
-		const randomIndex = Math.floor(Math.random() * charSet.length);
-		password += charSet[randomIndex];
-	}
-
-	return shuffleString(password);
-};
-
 const studentFirstLogin = (environment) => {
-	const newPassword = generateStrongPassword(12);
-	Cypress.env("password", newPassword);
+	Cypress.env("password", "SET_NEW_PWD_BY_STUDENT");
```

Comment:
```suggestion
	Cypress.env("password", Cypress.env("SET_NEW_PWD_BY_STUDENT"));
```
Otherwise you set it to the string "SET_NEW_PWD_BY_STUDENT", don't you?